site/pages/type-of-consequence/[disruptionId]/[consequenceIndex].page.tsx

```diff
@@ -72,6 +72,13 @@ const TypeOfConsequence = (props: ConsequenceTypePageProps): ReactElement => {
                             ) : (
                                 <></>
                             )}
+                            <Link
+                                role="button"
+                                href={"/api/type-of-consequence?draft=true"}
```

Comment:
Was it decided that we should leave this button here? If so it could just be a link back to the dashboard page

Reply:
the button is supposed to be removed